lib/companies_web/plugs/authorize.ex

```diff
@@ -10,7 +10,11 @@ defmodule CompaniesWeb.Plugs.Authorize do
 
   def init(opts), do: opts
 
-  def call(%{assigns: %{current_user: %{maintainer: true}}} = conn, _opts) do
+  def call(%{assigns: %{current_user: %{maintainer: true}}} = conn, maintainer: true) do
+    conn
+  end
+
+  def call(%{assigns: %{current_user: %{}}} = conn, _opts) do
```

Comment:
This pattern matching will allow access to the admin panel to all authorized users.
We need to check that the opts are empty.
```suggestion
  def call(%{assigns: %{current_user: %{}}} = conn, []) do
```

Reply:
Good catch!